src/Authenticator/JwtAuthenticator.php

```diff
@@ -126,11 +127,13 @@ public function getPayload(ServerRequestInterface $request = null)
         $payload = null;
         $token = $this->getToken($request);
 
-        if ($token) {
+        if ($token !== null) {
             $payload = $this->decodeToken($token);
         }
 
-        return $this->payload = $payload;
```

Comment:
Is this good style?
If not, could it be caught by some checker in future?

Reply:
It is not necessarily something of an issue.

Reply:
The previous way uses 2 fewer lines of code, and is readable in my opinion. This form is fine too though.

Reply:
One last thing that bugs me is that a getters modifies the instance property...